src/workflows/steps.js

```diff
@@ -521,22 +535,34 @@ export function askReposToUpdate( state ) {
 	} );
 }
 
-export function askVersion( pkg ) {
+export function askVersion( dependency ) {
+	const { pkg, version } = dependency;
 	return () => util.prompt( [ {
 		type: "input",
-		name: "version",
-		message: `What version do you want to update ${ pkg } to:`
-	} ] ).then( ( { version } ) => {
-		return Promise.resolve( { pkg, version } );
+		name: "vers",
+		message: `Update ${ chalk.yellow( pkg ) } from ${ chalk.yellow( version ) } to:`
+	} ] ).then( ( { vers } ) => {
+		return Promise.resolve( { pkg, version: vers } );
 	} );
 }
 
 export function askVersions( state ) {
-	const { packages } = state;
-	const prompts = packages.map( pkg => askVersion( pkg ) );
+	const { dependencies } = state;
+	const prompts = dependencies.map( dependency => askVersion( dependency ) );
+
+	return sequence( prompts ).then( deps => {
+		state.dependencies = deps;
+
+		const tagIdentifier = /^\d+\.\d+\.\d+\-(.+)\.\d+$/;
+		state.identifier = deps.reduce( ( memo, dep ) => {
```

Comment:
so we are combining all the prereleases identifiers into one if they are different?
i thought we talked about either prompting them or just picking the first one in that case